ably/rest/rest.py

```diff
@@ -41,6 +41,9 @@ def __init__(self, key=None, token=None, token_details=None, **kwargs):
           - `auth_callback`: Undocumented
           - `auth_url`: Undocumented
           - `keep_alive`: use persistent connections. Defaults to True
+          - `fallbackHostsUseDefault`: use default fallback servers
+            if connection to default server fails. Defaults to True
+          - `fallbackHosts`: list of fallback servers
```

Comment:
They are not fallback servers, they are fallback endpoints / hosts, but not servers.  Fiddly I know, but better to describe things correctly.